testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawITSupport.java

```diff
@@ -35,6 +36,7 @@
 /**
  * Support class for raw ITs.
  */
+@Category(RawTestGroup.class)
```

Comment:
IIRC contiperf was the only blocker to moving to junit 4.12 - looks like they cut a release with @cstamas' fix, but it's not been deployed anywhere that I could see. We could arrange to get that release onto central, or it might be easier to just remove contiperf from the one class in nexus that uses it (RecordIdObfuscatorPerfSupport)